requirements/bddtests.in

```diff
@@ -1,3 +1,4 @@
+pip-tools
```

Comment:
I don't know why I never thought of this before but if you need `pip-tools` to be installed in each venv then just add it to the `*.in` files! That's what they're for. Note that it's not added to `requirements.in` so it's not installed in production.